test/integration-tests/UniswapMixed.test.ts

```diff
@@ -516,6 +557,58 @@ describe('Uniswap V2 and V3 Tests:', () => {
       expect(ethBalanceAfter.sub(ethBalanceBefore)).to.eq(fullAmountOut.sub(gasSpent))
     })
 
+    it.only('ERC20 --> ERC20 split V4 and V4 different routes, with wrap, aggregate slippage', async () => {
+      // DAI -> USDC -> WETH
+      // and DAI -> USDC -> ETH -> WETH
```

Comment:
ooo ok this was a little confusing, I thought they were both routes INTO weth, but we are actually doing one into weth and one into eth and then a wrap, maybe just undo this last comment for the second router to weth